CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Allow multiple concurrent swaps with the same peer on the ASB.
   This is a breaking change because the swap ID is now agreed upon between CLI and ASB during swap setup.
   Resuming swaps started prior to this change can result in unexpected behaviour.
+- Quote protocol returns JSON encoded data instead of CBOR.
+  This is a breaking change in the protocol handling, old CLI versions will not be able to process quote requests of ASBs running this version.
+
+### Added
+
+- Websocket support for the ASB.
+  The ASB is not capable to listen on both TCP and Websocket connections.
+  Default websocket listening port is 9940, default TCP remains unchanged at 9939.
```

Comment:
These two lines are sort of contradicting each other, don't you think? Just mentioning the new port is good enough. I think it is common understanding that you can only have one transport protocol per port.

Reply:
That's because it should not be `not` but `now` 😅